packages/calypso-e2e/src/lib/pages/my-home-page.ts

```diff
@@ -4,7 +4,8 @@ const selectors = {
 	visitSiteButton: '.button >> text=Visit site',
 
 	// Task card (topmost card)
-	taskHeadingMessage: ( message: string ) => `div.task h2:has-text("${ message }")`,
+	taskHeadingMessage: ( message: string ) =>
+		`.primary__customer-home-location-content :text("${ message }")`,
```

Comment:
The previous selector was too restrictive and did not cover all scenarios.

eg. congratulatory message on launching the site is matched but not the message about updating the selected vertical's design to suit your needs.